core-web/apps/dotcms-ui/src/app/portlets/dot-containers/dot-container-create/dot-container-code/dot-container-code.component.scss

```diff
@@ -1,7 +1,7 @@
 @use "variables" as *;
 
 .dot-container-code__content-tab-container {
-    padding: $spacing-4;
+    padding: 0 $dot-container-spacing 0 $dot-container-spacing;
```

Comment:
Use the closing spacing variable to this... sorry the design didn't match any of the spacing variables.

Reply:
Noted

Reply:
updated